README.md

```diff
@@ -131,9 +131,124 @@ You can add your customized Python code to process your input and output data. T
 1. The customized Python code file should be named `inference.py` and it should be under `code` directory of your model archive.
 2. `inference.py` should implement either a pair of `input_handler` and `output_handler` functions or a single `handler` function. Note that if `handler` function is implemented, `input_handler` and `output_handler` will be ignored.
 
+To implement pre/post-processing handler(s), you will need to make use of the `Context` object created by Python service. The `Context` is a `namedtuple` with following attributes:
+- `model_name`: the name of the model you will to use for inference, for example 'half_plus_three'
+- `model_version`: version of the model
+- `method`: inference method, for example, 'predict', 'classify' or 'regress'
+- `rest_uri`: the TFS REST uri generated by the Python service
+- `grpc_uri`: the GRPC uri generated by the Python service
+- `custom_attributes`: content of 'X-Amzn-SageMaker-Custom-Attributes' header from the original request
+- `request_content_type`: the original request content type, defaulted to 'application/json' if not provided
+- `accept_header`: the original request accept type, defaulted to 'application/json' if not provided
+- `content_length`: content length of the original request
```

Comment:
Let's add an example to each item above.

Reply:
done, except that I didn't give example to 'content_length'